App/Sagas/TextileEventsSagas.ts

```diff
@@ -19,11 +34,140 @@ function displayNotification(message: string, title?: string) {
   })
 }
 
-// export function * runBackgroundUpdate() {
-// This used to be some exported function from the SDK
-// TODO: Remove all this, make sure background triggers are handled in native SDKs
-// yield call(BackgroundTask)
-// }
+function nodeEvents() {
+  return eventChannel<
+    | GroupAction
+    | PhotosAction
+    | PhotoViewingAction
+    | ContactsAction
+    | DeviceLogsAction
+    | NotificationsAction
+    | TextileEventsAction
+  >(emitter => {
+    const subscriptions: EventSubscription[] = []
+    subscriptions.push(
+      Textile.events.addThreadUpdateReceivedListener(update => {
+        const { type_url } = update.payload
+        if (
+          type_url === '/Text' ||
```

Comment:
Might it be better to put these type strings into a list, like `refreshTypes`, and check if `type_url` is in that list?

Reply:
Possibly something like that. It would take multiple lists because we're matching against a unique set each time. But anyway, this code was copy/pasted from another file, and I want to keep this PR focused on the task at hand. We can revisit how to deal with these protobuf `Any` types as another task (the answer is probably expanding the callbacks in the SDK layer so we don't pass `Any` objects in here).